stable/database/values.yaml

```diff
@@ -357,6 +360,23 @@ database:
   isManualVolumeProvisioning: false
   isRestore: false
 
+
 openshift:
   # change this to true if you want to deploy using OpenShift proprietary features
   enabled: false
+
+insights:
+  # disable insights by setting insights.enabled=false or admin.insights="" or admin.insights=null
+  # enabled here as a super values file that works for both admin and database helm charts can set
+  # to false to disable for both helm charts.
+  enabled: false
+  image:
+    registry: docker.io
+    repository: nuodb/nuocd
```

Comment:
this repo does not exist

Reply:
yes,  I know that.  Which is why sidecar is disabled for now.  You can enable sidecar and build nuodb/nuocd from nuodb-dashboards-influx and push to a local registry.  Then override image.registry.   If you or someone else can push to github nuodb then go ahead and build and install.